examples/rapids/mlflow_project/README.md

```diff
@@ -0,0 +1,39 @@
+### Train and Publish Locally With RAPIDS and MLFlow
+**[RAPIDS](https://rapids.ai/)** is a suite of open source libraries for GPU-accelerated analytics. 
+
+**[RAPIDS cuML](https://github.com/rapidsai/cuml)** matches the scikit-learn API, so it can build on MLFlow's existing support for scikit-learn-like models to support 
+persistence and deployment."
+
```

Comment:
Maybe add a sentence describing what our example workflows do, e.g:

```suggestion
The example workflows below train RAPIDs regression models to predict airline flight delays, using MLflow to log models and deploy them as local REST API endpoints for real-time inference.
```

^assuming that such a description is accurate :)